### PR TITLE
Add safe testing, Fix tests follow changes to file structure and naming

### PR DESCRIPTION
**Pull request by natebuch**

### CODE REVIEW

File: tests/extensions/sde009-safe_test.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import { ExecutorDao } from '../models/executor-dao-model.ts';
+import { SDE006Membership, SDE006_MEMBERSHIP_CODES } from '../models/sde006-membership-model.ts';
+import { SDE007ProposalVoting, SDE007_PROPOSAL_VOTING_CODES } from '../models/sde007-proposal-voting-model.ts';
+import { SDE008ProposalSubmission, SDE008_PROPOSAL_SUBMISSION_CODES } from '../models/sde008-proposal-submission-model.ts';
+import { SDE009Safe, SAFE_CODES } from '../models/sde009-safe-model.ts'; 
+import { PROPOSALS, EXTENSIONS, TEST_EXTENSIONS, TEST_PROPOSALS } from '../utils/contract-addresses.ts';
+
+
+Clarinet.test({
+  name: '😢 SDE009Safe - Token not whitelisted',
```
Comment (ryanwaits) [inline tests/extensions/sde009-safe_test.ts:16]: this doesnt really check if a token is not whitelisted, this is an authentication check on who is allowed to call setWhitelisted

**Review by ryanwaits — COMMENTED** [review]
looks good! a couple minor notes:

- [ ] `sdp007-whitelist-asset` should be changed where necessary to `sdp008-whitelist-asset` so it doesn't clash with an existing sdp007
- [ ] `sdp008-whitelist-asset` should be added below `sdp007-add-signer` in the Clarinet.toml

File: tests/extensions/sde009-safe_test.ts (hunk 1 above)
Comment (natebuch) [reply tests/extensions/sde009-safe_test.ts:16]: sdp007-whitelist-asset was an old file that was removed.   Removed reference in Clarinet.toml

In /tests/proposals, sdp009-test-whitelist-asset is the current file as an 008 exists already.  Was trying to discern/follow your paradigm there.  Can do either of the below:
1) Move it out of tests and rename with sdp008...
2) Leave in tests as 009

Moved current /tesets/proposals 009 to be below 008 in Clarinet.toml

Comment (natebuch) [reply tests/extensions/sde009-safe_test.ts]: SD009Safe - Token not whitelisted: 
Modified name of test to more accurately reflect what it is testing for.

Added actual whitelist check at beginning of whitelist asset testing to confirm asset is not initially whitelisted.

**Pull request closed without merging**